scalafix-core/shared/src/main/scala/scalafix/internal/config/CustomMessage.scala

```diff
@@ -0,0 +1,29 @@
+package scalafix.internal.config
+
+import metaconfig.{Conf, ConfError, ConfDecoder, Configured}
+import org.langmeta._
+import MetaconfigPendingUpstream.XtensionConfScalafix
+
+import scala.language.implicitConversions
+
+case class CustomMessage[T](value: T, message: Option[String])
+object CustomMessage {
+  def decoder[T](field: String)(
+      implicit ev: ConfDecoder[T]): ConfDecoder[CustomMessage[T]] =
+    ConfDecoder.instance[CustomMessage[T]] {
+      case obj: Conf.Obj =>
+        (obj.get[T](field) |@| obj.get[String]("message")).map {
+          case (value, message0) => {
+            val message =
+              if (message0.split(System.lineSeparator).size > 1) {
```

Comment:
System.lineSeparator will produce inconsistent behavior across OS since `.scalafix.conf` contains the same bytes, we should use "\n" instead. `.split().size` can also be replaced with `"".contains('\n')`